utils.c

```diff
@@ -665,3 +665,13 @@ label_exec (const char *exec_label)
 #endif
   return 0;
 }
+
+int
+label_valid (const char *label)
+{
+#ifdef HAVE_SELINUX
+  if (is_selinux_enabled () > 0 && label)
+    return security_check_context ((security_context_t)label);
+#endif
+  return -1;
```

Comment:
This function should only be called if `label_support()` succeeds, so I don't think we need to re-check `is_selinux_enabled`, right?